answers.md

```diff
@@ -1 +1,18 @@
 # Answers Go In This File
+//Shaina 
+//Qustion 1
+1. Git is a tool used by software developers to manage their code. Git keeps track of code by saving existing code. Software developers use it to code with others by branching, merging commits, and adding their code to the main base. Git is a local repository.
```

Comment:
This has so many great elements. I love that you are calling out that git is used to manage code and saves existing code and you are calling out some key actions used with Git like branching and merging commits. 

The only thing I would update is the last sentence "Git is a local repository". I see what you're going for but a more correct way of putting this would be "Git is the command-line tool used to track and commit changes made to a local repository, or upload commits to a remote repository". 

To be clear: The repository on Github is the remote repository. The folder that is created when we `git clone` a remote repository is a local repository. Git is the tool that let's us manage repositories using the command line.